### PR TITLE
Max tweaks

### PR DESCRIPTION
**Pull request by omgmog**
A bunch of stuff, it's pretty cool.

### CODE REVIEW

File: _hacks/10.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+hack_number: 10
+date: 2021-01-30
+happened: true
+---
+We had our most _international_ day of remote hacks yet, with participants California and Munich!
+
```
Comment (lpmi-13) [inline _hacks/10.md:7]: Participants in/from?

File: _hacks/10.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+### Things people did
+
+- **Ben** pulled **everyone** down the rabbit hole of LIDAR/PCAP and also some AWS hacks.
+- **Max** intended to work on his LensLok solver, but instead worked on the Airquotes the Podcast Airquotes _branding_
+- **Panda** worked on some software to help keep his plants alive.
+- **Dan** learned some Typescript.
+- **Ruth** etched a design and cut out a jigsaw using her laser cutter, and also hacked a bit on some WebAudio convolution.
```
Comment (benfoxall) [inline _hacks/10.md:14]: Ah, I plonked down some notes the other day.

I've mashed them in (might need some tweaking or more info):

```suggestion
- **Henry** joined us from California, where it was 2am! Simultaneously becoming our remote-ist and our most nocturnal hacker.
- **Ruth** showed people her laser cutter, which she used to etch & cut a jigsaw 🧩⚡️. Then she did some hacking with convolution filters using WebAudio 🎷.
- **Dimitri** was planning to _learn_ **Rust** by using it to compile **Elm** to **JavaScript**.  Though he ended up helping people find bugs before they'd even written the code that introduced them 💅.
- **Panda** planned to write some software to help keep his plants alive, but instead got sucked into Bens AWS vortex (🌱💀)
- **Max** intended to work on his LensLok solver, but instead worked on the Airquotes the Podcast Airquotes _branding_
- **Mike** used NextJS and MDX to build a blog with dynamic graphs in it!
- **Dan** learned some Typescript, then taught people aws, dns & service features, and then became an elite streamer by mastering OBS
- **Ben** pulled **Dan**, **Dimitri**, **Ryan**, **Panda** and **everyone** down the rabbit hole of LIDAR/PCAP and AWS hacks. (he's very sorry, but also had a great time.)
- **Alan** got to grips with static site building with 11ty 👨‍💻
- **Emanuil** joined us in the afternoon, and worked on a FOSSJobs site 👍 
- **Piers** told us about his VW Camper hack and chatted about augmented reality, then programmed a microcontroller to measure distances.
- **Adam** intended to create some statically rendered flashcards with nextJS but ended up just interviewing people and getting a PR Merged into Ruth's generative artistry website
```

File: assets/style.css [hunk 3]
```diff
@@ -130,12 +130,13 @@ ol li:before {
...
 
-.📅 time {
+.📅 .⏰,
+.📅 .☹️ {
   font-size: 3em;
   display: block;
   font-weight: inherit;
 }
-.📅 time > span {
+.📅 .⏰ > span {
```
Comment (benfoxall) [inline assets/style.css:139]: `.💅 `

**Review by benfoxall — APPROVED** [review]
@omgmog this is amazing.  It's always such a faff to update the index page, but you've totally fixed that. Great work!

I've added some more hacks (might be more to add that i've missed (@spikeheap 
@lpmi-13 ?)).  Other than that 🚢

File: _hacks/10.md (hunk 2 above)
Comment (omgmog) [reply _hacks/10.md:14]: 💪

File: _hacks/10.md (hunk 1 above)
Comment (omgmog) [reply _hacks/10.md:7]: You and your fancy prepositions! Right I'll fix that, I may have been mashing the keyboard too hard..

File: _hacks/10.md (hunk 2 above)
Comment (omgmog) [reply _hacks/10.md]: I'll take your changes (need to commit it manually 😓 )

Comment (omgmog) [reply _hacks/10.md]: Resolved in 0880a2f

File: assets/style.css (hunk 3 above)
Comment (omgmog) [reply assets/style.css:139]: 💁

**Comment by omgmog** [comment]
Okay cool cool cool, I don't have write access to merge this, but glad you're all happy with the changes 🎁

**Comment by omgmog** [comment]
Oh and if you prefer, rather than using the Jekyll `redirect-from` plugin to generate the redirect in 2524288d, we could probably use [Netlify redirects](https://docs.netlify.com/routing/redirects/)?

I've never used Netlify and only saw that capability in the deploy preview stuff 🤔 

The `redirect-from` plugin just generates an html file at the old URL with a meta refresh afaik, rather than doing a proper 301/302.

**Comment by lpmi-13** [comment]
oh wild, thought you had write access...lemme see about that...

**Comment by lpmi-13** [comment]
alright, Sound Guy's transition to being the boss of remotehack town is complete...MERGE THAT!

**Pull request merged**